.clj-kondo/hooks/clojure/core.clj

```diff
@@ -179,6 +179,134 @@
     (non-thread-safe-form-should-end-with-exclamation* node))
   {:node node})
 
+;; ----------------- Modules -----------------
```

Comment:
It would be nice to hide all these helper function in a (testable) namespace, but I didn't figure out how to do that.

Reply:
Speaking of tests, I didn't find an easy way to jack into this kondo namespace with Cursive, so I didn't bother with "rich comment" tests either. I tested manually by inserting actual requires in the app and running the linter but don't have a great way to share that here. I guess I could show it via screenshots or a loom, but this seems a bit OTT.